gatsby-node.js

```diff
@@ -4,19 +4,24 @@
  * See: https://www.gatsbyjs.org/docs/node-apis/
  */
 const path = require('path')
-const { createOpenGraphImage } = require("gatsby-plugin-open-graph-images");
+const { createOpenGraphImage } = require('gatsby-plugin-open-graph-images')
 
 const createPageDataStruct = require('./gatsbyNodeHelper/index')
 const houseLegislators = require('./src/data/house_legislators.json')
 const senateLegislators = require('./src/data/senate_legislators.json')
+const { getLegislatorUrlParams } = require('./src/utilities')
 
 const makePage = ({ chamber, pageData, createPage, legislatorId }) => {
-  const ogImageFilename =
-    (pageData.legislator.name + '-' + pageData.legislator.district)
+  const ogImageFilename = (
+    pageData.legislator.name +
+    '-' +
+    pageData.legislator.district
```

Comment:
This could probably be replaced with your new `getLegislatorUrlParams(pageData.legislator)` for consistency, but not a big deal.